maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -245,9 +242,11 @@ private void setupBuildRepos(BuildExecution execution, Indy indy)
         if (!indy.stores().exists(groupKey)) {
             // if the product-level storage repo (for in-progress product builds) doesn't exist, create it.
             StoreKey hostedKey = new StoreKey(MAVEN_PKG_KEY, StoreType.hosted, buildContentId);
+            boolean tempBuild = execution.isTempBuild();
+			boolean snapshotBuild = tempBuild && StringUtils.isEmpty(execution.getTempBuildTimestamp());
```

Comment:
Not all temporary builds are builds with timestamp. It's just one of the options.
So, it should be:
boolean snapshotBuild = tempBuild

And ehm, formatting.

Reply:
Argh, sorry, stupid new eclipse, that tabs vs spaces irritates me.

Regarding snapshot build isn't it a temporary build which has an empty timestamp?

Reply:
Temporary build is a build with the flag tempBuild.

But it doesn't say anything about the alignment.
It can be normal snapshot, it can be with timestamp, it can have standard rh suffix and nothing else, there can be mix of it. We basically don't care and it's up to the user to configure it. So, we're just saying, we will also allow snapshots as it doesn't hurt us.
And for the timestamp build it's expected that there will be no snapshots as we change it in the alignment (if it's not affected somehow by PME params), but for the builds without timestamp, it's more probable the snapshots could theoretically be there.
Basically, we don't know it on this level and we don't have to, so for all the temp builds, we will just allow the snapshots.

And btw. the timestamp is not empty, it's null.